CONTRIBUTING.md

```diff
@@ -165,6 +179,86 @@ This draft pull request signals the others that someone is working on this featu
+If the commit message itself needs to be changed, use `git commit --squash <sha1 of commit to fix> -e`.
+Below the subject line, add a second subject line, followed by the complete commit message and the Signed-off-by.
+During the autosquash rebase, we'll only retain the second commit message (see below).
+
+In the pull request's web interface, mark the comments that have been addressed as Resolved.
+Also mark the comments that are optional as Resolved.
+If you don't take the suggestion directly or you disagree or only apply it in part, don't mark it as Resolved.
+Regularly push your branch.
+
+There may be several iterations in the review.
+Finally, the pull-request is marked as Approved by the maintainers.
+However, it cannot be applied as is because the commits are still "dirty".
+The last step is to perform `git rebase -i --autosquash master`.
+This will apply the fixup commits automatically.
+For the squash commits, it will stop in an editor to allow you to update the commit message.
+Just use the last commit message, remove all the rest.
+Finally, force-push the branch.
+The DCO check will now succeed and the pull request is ready for merging.
+
+Note that there are a few cases where it is not possible to use fixup commits.
+In these cases, use rebase and create a clean series of commits again.
+Add a comment in the pull request why this was done.
+This is needed at least in the following cases:
+* If you need to pull in changes from master or another branch or pull request.
+* If you need to reorder the commits.
+* If you need to move a subset of the changes from one commit to another commit.
+
+If you are working on a big feature, you often encounter something small that needs to be fixed or improved that is relatively independent of the feature.
+Such a change can be included as one of the first commits in the pull request.
+However, often it's useful to create a separate pull request for it, so it can be applied more quickly.
+The typical workflow for this is:
+* Make the fix and commit it with a proper commit message.
+* Check out a new branch `hotfix/<description>` based on master.
+* Cherry-pick the fix commit.
+* Push and create a pull request.
+* Check out the development branch and rebase on the hotfix branch. This will automatically remove th fix commit.
+
+### Definition of done
+
+Before a pull request can be merged, it must be considered "Done".
+That means the following conditions must hold.
+* All commits have a Signed-off-by. Automatic with the DCO check.
+* At least one maintainer has reviewed and approvied.
+* Code builds on Ubuntu, with `MSGLIB=zmq` and `BWL_TYPE=DUMMY`. Automatic with Travis CI.
+* Unit tests run successfully. Automatic with Travis CI.
+* Controller and agent go to operational. Manual, by starting `local_bus`, `ieee1905_transport`, `beerocks_controller` and `beerocks_agent` and verifying that the agent log reports OPERATIONAL.
```

Comment:
Consider changing this to using the prplmesh_utils.sh instead (assuming it gets merged soon).
```suggestion
* Controller and agent go to operational. Manual, by starting `local_bus`, `ieee1905_transport`, `beerocks_controller` and `beerocks_agent` using `sudo ./install/scripts/prplmesh_utils.sh start` and verifying that the agent log reports OPERATIONAL.
```

Reply:
nevermind I updated the readme in #55